BUILD.rst

```diff
@@ -0,0 +1,195 @@
+=====================================
+Cask Data Application Platform - CDAP
+=====================================
+
+**Standalone and Distributed CDAP**
+
+Building CDAP Maven
+===================
+
+Clean all modules
+-----------------
+
+  mvn clean
+
+Run all tests, fail at the end
+------------------------------
+
+  MAVEN_OPTS="-Xmx1024m -XX:MaxPermSize=128m" mvn test -fae
+    
+Build all modules
+-----------------
+
+  mvn package
+
+Run checkstyle, skipping tests
+------------------------------
+
+  mvn package -DskipTests
+
+Build a particular module
+-------------------------
+
+  mvn package -pl [module] -am
+
+Run selected test
+-----------------
+
```

Comment:
Yes, I probably should have. What did you want to see that changed? This particular file?